src/bowling.js

```diff
@@ -0,0 +1,49 @@
+class Bowling {
+  constructor() {
+    this.rolls = [];
+  }
+
+  roll(pins) {
+    this.rolls.push(pins);
+  }
+
+  score() {
+    let result = 0;
+    let rollIndex = 0;
+    let game = this;
+
+    for (let frame = 0; frame < 10; frame++) {
```

Comment:
It would be good to store 10 in a constant somewhere, so that it's clear what its meaning is.